package.json

```diff
@@ -111,7 +119,7 @@
     "clean:install": "npm run clean && npm install",
     "clean:obsolete-snapshots": "npm test -- -u",
     "compile": "npm run compile:clean && npm run compile:build",
-    "compile:build": "lerna ls | sed 's/@cerner\\///g' | cut -d ' ' -f1 | xargs -I {} babel packages/{}/src --out-dir packages/{}/lib --copy-files",
+    "compile:build": "lerna ls -all | sed 's/@cerner\\///g' | cut -d ' ' -f1 | xargs -I {} babel packages/carbon-graphs/src --out-dir packages/{}/lib --copy-files",
```

Comment:
Can you describe what this change is doing?

Reply:
Sorry, this change has a mistake. Here is the actual change, made in order to resolve an error caused by the new terra-graphs-docs package not having a src folder: https://github.com/cerner/terra-graphs/pull/105/commits/e56c985de4eb413da56329bb62d281911c4981bd